### DISCUSSION TITLE
My posts won't show up

### DISCUSSION THREAD

**Discussion by ghost** [community/community#39599] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello,
I have uploaded my md file in _posts folder in my repo but it shows file not found.

My site is https://sndn.github.io/

Thanks

**Comment by airtower-luna** [comment, marked as answer]
The site seems to load fine as far as I can see, including the two posts. Where exactly do you have trouble?

**Reply by ghost** [reply]
Strange.... The second post is saying 404 file/page not found. 
I have even cleared my cookies.

**Reply by radar-blip** [reply]
Site is loading fine for me too. Both links are also working.

**Reply by ghost** [reply]
Thank you Luna and Lawson.
Its working now. Sometimes Jekyll or Github acts weird.

**Reply by radar-blip** [reply]
You're welcome! Glad its working for you!
